CondFormats/CTPPSReadoutObjects/src/classes_def.xml

```diff
@@ -20,4 +20,16 @@
     <class name="CTPPSPixelGainCalibration::DetRegistry"/>
     <class name="std::vector<CTPPSPixelGainCalibration::DetRegistry>"/>
     <class name="CTPPSPixelGainCalibrations" class_version="0"/>
+
+    <class name="PPSTimingCalibration::Key"/>
+    <class name="std::map<PPSTimingCalibration::Key,std::vector<double> >"/>
+    <class name="std::map<PPSTimingCalibration::Key,std::pair<double,double> >"/>
+    <class name="std::pair<PPSTimingCalibration::Key,std::vector<double> >"/>
+    <class name="std::pair<PPSTimingCalibration::Key,std::pair<double,double> >"/>
+    <class name="PPSTimingCalibration" class_version="0">
+      <field name="formula_"/>
+      <field name="parameters_" mapping="blob"/>
+      <field name="timeInfo_" mapping="blob"/>
```

Comment:
... and same for this field.

Reply:
@forthommel 
same as before, no longer used in the persistency